### ISSUE TITLE
Remove item button missing

### ISSUE THREAD

**Issue by mlaxwong** [unclead/yii2-multiple-input#183]
Hi, I really like yii2-multiple-input, on my application everything working will version 2.10.0, but after update to version 2.11.1 all the item remove button have missing. 

Is my config problem? Or mission any config option?

```php
MultipleInput::widget([
	'name' => $this->name,
	'addButtonPosition' => MultipleInput::POS_FOOTER,
	'allowEmptyList' => true,
	'data' => $this->_data,
	'addButtonOptions' => [
		'class' => 'btn btn-default btn-sm'
	],
	'removeButtonOptions' => [
		'class' => 'btn btn-danger btn-sm',
		'label' => '<i class="fa fa-trash"></i>'
	],
	'columns' => [
		[
			'name' => 'name',
			'options' => ['placeholder' => 'Name']
		],
		[
			'name' => 'setting',
			'options' => ['placeholder' => 'Setting']
		],
	]
]);
```

**Comment by unclead** [comment]
thx for reporting a bug
fixed via 05ab8ba3ae7682204c988e3288c8b8d6110a902e

**Comment by mlaxwong** [comment]
Thanks

**Comment by gitrequests** [comment]
Hi. With this fix POS_ROW don`t render add button, and render 2 buttons (begin and end positions) with POS_ROW_BEGIN!
Is my config problem? Or mission any config option?

**Comment by unclead** [comment]
@gitrequests  i hope it fixed now (06bc83b0508b0a9ec9de8622d6cae2433d9bac45)
